classroom_spec.rb

```diff
@@ -0,0 +1,18 @@
+require './classroom'
+require './student'
+
+describe Classroom do
+  let(:classroom) { Classroom.new('English') }
+  let(:student) { Student.new(18, 'Grammerly', parent_permission: true) }
+
+  it 'should initialize correctly' do
+    expect(classroom.label).to eq('English')
+    expect(classroom.students).to be_empty
+  end
+
+  it 'should add a student' do
+    classroom.add_student(student)
+    expect(classroom.students).to include(student)
+    expect(student.classroom).to eq(classroom)
+  end
+end
```

Comment:
- [ ] This class was tested after submitting for review which is not a professional action. You are like asking a reviewer to restart the review process again, pulling changes and verify if everything works :angry: Please always wait for a review before working on changes :+1: The next reviewer will double check if everything works well